From: Morose <chenlinxiang@kylinos.cn>
Date: Sun, 9 Jun 2024 15:46:11 +0800
Subject: Update

---
 types/wlr_linux_dmabuf_v1.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/types/wlr_linux_dmabuf_v1.c b/types/wlr_linux_dmabuf_v1.c
index d018d77..1a8bbc6 100644
--- a/types/wlr_linux_dmabuf_v1.c
+++ b/types/wlr_linux_dmabuf_v1.c
@@ -215,7 +215,6 @@ static bool check_import_dmabuf(struct wlr_linux_dmabuf_v1 *linux_dmabuf,
 		}
 		if (drmCloseBufferHandle(linux_dmabuf->main_device_fd, handle) != 0) {
 			wlr_log_errno(WLR_ERROR, "Failed to close buffer handle");
-			return false;
 		}
 	}
 	return true;
